apps/demo/src/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
Looks like we have 2 `index.html`, is this expected?

Reply:
fixed